apps/web/components/ui/HeroContent/HeroContent.vue

```diff
@@ -11,15 +11,22 @@
     </div>
 
     <div class="p-4 md:p-10 md:max-w-[768px] md:flex md:flex-col md:justify-center md:items-start md:basis-2/4">
-      <p class="typography-text-xs md:typography-text-sm font-bold tracking-widest text-neutral-500 uppercase">
+      <p
+        class="typography-text-xs md:typography-text-sm font-bold tracking-widest uppercase"
+        :class="heroItemProps?.taglineColor"
```

Comment:
**question**: Why do you need the condition on `heroItemProps` here, given that we don't need that for the texts themselves?

Reply:
You mean why I've added the optional parameter? ("?")

Reply:
Yes. It seems to me that we'll always have `heroItemProps` in some capacity, so this check would be unnecessary?

Reply:
I assumed that the 3 colors can be optional in this case. So basically if there is no taglineColor key in the JSON it will not try to add anything here.